routers/posts.py

```
@@ -0,0 +1,228 @@
+    if post["user_id"] != current_user["id"]:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="작성자만 삭제할 수 있습니다"
+        )
+
+    comments = data.load_data("comments.json")
+    comments = [c for c in comments if c.get("post_id") != post_id]
+    data.save_data(comments, "comments.json")
+
+    likes = data.load_data("likes.json")
+    likes = [l for l in likes if l.get("post_id") != post_id]
```

Ruff (0.14.11)

Ambiguous variable name: `l`

(E741)